### ISSUE TITLE
Change order of Rating Yes, no, cancel links

### ISSUE THREAD

**Issue by Divyaadz** [kobakei/Android-RateThisApp#70]
How to change the order of Rating Yes, no, cancel links in the dialog box?

**Comment by AdityaAnand1** [comment]
Yes! The "No, Thanks" should go to the left and "Later" in it's place.

I reckon this way users would be more likely to click "Later" instead of "No"
